broker-util/oo-admin-chk

```diff
@@ -169,9 +169,11 @@ node_hash.each do |gear_uuid, gear_info|
   datastore_gear_info = $datastore_hash[gear_uuid]
   if !datastore_gear_info
     if !datastore_has_gear?(gear_uuid)
+      server_identity = gear_info[0]
       puts "#{gear_uuid}...FAIL" if $verbose
       print_message "Gear #{gear_uuid} exists on node #{gear_info[0]} (uid: #{gear_info[1]}) but does not exist in mongo database"
```

Comment:
Since you defined `server_identity` as `gear_info[0]`, you should replace `#{gear_info[0]}` here with `server_identity` as to not duplicate code.